patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -688,12 +688,30 @@
 +        return builder.build();
 +    }
 +
++    /**
++     * Checks whether an item can be put into a container that is able to retain its contents as an item
```

Comment:
I think this should be
`Returns whether this item can be stored in an item container (backpacks, Shulker boxes, etc).`